results/CLionResults.md

```diff
@@ -2102,6 +2102,10 @@ R1517 |
 R1518 |
 R1519 |
 R1520 |
+R1521 |
```

Comment:
You'll need to rebase on HEAD as I'm adding lots of tests recently.  Easiest thing to do is to mark the tests as newly added and re-add them using the `add-tests` tool as described in `Contributing.md`